common/supplemental/metaZones.xml

```diff
@@ -404,8 +404,8 @@ For terms of use, see http://www.unicode.org/copyright.html
 				<usesMetazone from="1988-10-30 02:01" mzone="Atlantic"/>
 			</timezone>
 			<timezone type="America/Grand_Turk">
-				<usesMetazone to="2015-11-01 06:00" mzone="America_Eastern"/>
-				<usesMetazone to="2018-03-11 07:00" from="2015-11-01 06:00" mzone="Atlantic"/>
+				<usesMetazone to="2015-03-08 07:00" mzone="America_Eastern"/>
+				<usesMetazone to="2018-03-11 07:00" from="2015-03-08 07:00" mzone="Atlantic"/>
```

Comment:
@yumaoka - Curious the reason for the change here?  The previous date seems more correct with regard to name changes.
https://www.timeanddate.com/time/zone/@3576993?year=2015